test/sample-test/sample_test_launcher.py

```diff
@@ -43,11 +46,22 @@ def __init__(self, test_name, results_gcs_dir, target_image_prefix='',
     # Capture the first segment after gs:// as the project name.
     self._bucket_name = results_gcs_dir.split('/')[2]
     self._target_image_prefix = target_image_prefix
+    self._is_notebook = None
     self._namespace = namespace
     self._sample_test_result = 'junit_Sample%sOutput.xml' % self._test_name
     self._sample_test_output = self._results_gcs_dir
     self._work_dir = os.path.join(BASE_DIR, 'samples/core/', self._test_name)
 
+  def _copy_result(self):
+    """ Copy generated sample test result to gcs, so that Prow can pick it. """
+    print('Copy the test results to GCS %s/' % self._results_gcs_dir)
+
+    utils.upload_blob(
+        self._bucket_name,
+        self._sample_test_result,
+        os.path.join(self._results_gcs_dir, self._sample_test_result)
+    )
+
   def check_result(self):
```

Comment:
Looks like this function is not used anymore. Should we remove it?

Reply:
Done. Thanks for pointing this out.